### ISSUE TITLE
Remove the fridge picture on the home page 

### ISSUE THREAD

**Issue by mohit200008** [mohit200008/FoodSaver20008#8]
Below the navbar there should be a carousel  showing pictures related to food and fridge. Below the carousel The Food about to expire section should be there in horizontal alignment.

**Comment by shefali12-ab** [comment]
Hello sir, Can I work on this issue?

**Comment by mohit200008** [comment]
yeah sure @shefali12-ab  go ahead

**Comment by shefali12-ab** [comment]
@mohit200008 sir , please tell me how many carousel slides I will add, also there are no images in a project so I am adding images on my own. you can also suggest images which I should add.

**Comment by mohit200008** [comment]
You can add images you feel, and there can be 3 to 4 slides. Also the the cards below the carousel must be in  horizontal direction.

**Comment by shefali12-ab** [comment]
https://drive.google.com/file/d/1L_bZOL1M9JZoQH1BA9QkCPF5jA6O-w_U/view?usp=sharing
@mohit200008 sir i have made a carousel card, please have a look.

**Comment by mohit200008** [comment]
Its good @shefali12-ab , Redesign the food about to expire section too

**Comment by mohit200008** [comment]
And that should be below the carousel

**Comment by shefali12-ab** [comment]
@mohit200008 okay sir
https://www.figma.com/file/gJTPQNhJJppwdim7LcVdzm/Untitled?node-id=0%3A1
check this design you want me to design like this??

**Comment by mohit200008** [comment]
> @mohit200008 okay sir https://www.figma.com/file/gJTPQNhJJppwdim7LcVdzm/Untitled?node-id=0%3A1 check this design you want me to design like this??

yes and make sure that the background goes with overall design of the page.Rest everything is good

**Comment by mohit200008** [comment]
![screencapture-food-saver20008-herokuapp-2022-02-22-00_37_18](https://user-images.githubusercontent.com/56565813/155013966-e01c26a4-5c64-4578-a24e-b83b5983e82d.png)
@shefali12-ab  the carousel images are in bad shape and they're over flowing. please resolve this issue to get the points.

**Comment by shefali12-ab** [comment]
@mohit200008 okay sure sir, can you connect on meet ?. As when I have done it, there is no such thing like overflowing.

**Comment by mohit200008** [comment]
We can connect in the evening

**Comment by mohit200008** [comment]
7 o clock is ok ?

**Comment by shefali12-ab** [comment]
yeah sure

**Comment by mohit200008** [comment]
Connect with me on discord we will have a meet

**Comment by mohit200008** [comment]
harvey#5191

**Comment by shefali12-ab** [comment]
i have messaged you, please check

**Comment by ankushj0801** [comment]
Can I work on this?
